rest/taskana-rest-spring/src/main/java/pro/taskana/common/rest/ldap/LdapClient.java

```diff
@@ -214,6 +214,46 @@ public AccessIdRepresentationModel searchAccessIdByDn(final String dn) {
     return accessIds;
   }
 
+  /**
+   * Validates a given AccessId / name.
+   *
+   * @param name lookup string for names or groups
+   * @return whether the given name is valid or not
+   */
+  public boolean validateAccessId(final String name) {
```

Comment:
I think we should either add some debugging here to understand why the validation fails, or we should throw an Exception with the reason stated in the message.